crawl-ref/source/map-knowledge.h

```diff
@@ -41,3 +41,10 @@ map_feature get_cell_map_feature(const map_cell& cell);
 bool is_explore_horizon(const coord_def& c);
 
 void reautomap_level();
+
+/**
+ * @brief Get the bounding box of the known map.
+ *
+ * @return vector of [topleft coord, bottomright coord] of bbox.
+ */
+FixedVector<coord_def, 2> known_map_bounds();
```

Comment:
`pair<coord_def, coord_def>` seems like a more fitting type to me. Did you reject this intentionally because you think a `FixedVector<2>` is better? If so I'm happy to hear why.

Reply:
No reason, really; my other consideration was a tuple but they're annoying to work with before C++17. Either way seems fine to me.

Reply:
changed and pushed; thank you